src/views/GovernanceReviewTeam/Actions/ActionForm.tsx

```diff
@@ -203,18 +272,46 @@ const ActionForm = ({
           )}
         />
 
+        {modal && (
+          <Modal isOpen={modalIsOpen} closeModal={() => setModalIsOpen(false)}>
+            <ModalHeading>{t(modal.title)}</ModalHeading>
+            {typeof modal.content === 'string'
```

Comment:
This switch to use a string as translation text is a bit tucked away. I think it should be simplified here so that it's just `{modal.content}`. And then use the translation function when setting the props.

If not maybe at least comment on the behavior in the ActionFormProps type property for modal.content, or name a new property so that the behavior is more noticeable.

Reply:
Actually, just a comment or property name to suggest the behavior is good. I misread this as being in the modal component, so I was wrong about the coupling issue.